src/index.html

```diff
@@ -1,13 +1,500 @@
+
+              <div class="powder__eco">
+                <ul class="eco__list">
+                  <li class="eco__item">
+                    <img
+                      class="eco__img"
+                      src="images/icons/no_synthetics.svg"
+                      alt="No synthetics and microplastics">
+
+                    <p class="eco__explanation">
+                      No synthetics and microplastics
+                    </p>
+                  </li>
+
+                  <li class="eco__item">
+                    <img
+                      class="eco__img"
+                      src="images/icons/recycled.svg"
+                      alt="product packaging can be recycled or reused">
+
+                    <p class="eco__explanation">
+                      product packaging can be recycled or reused
+                    </p>
+                  </li>
+
+                  <li class="eco__item">
+                    <img
+                      class="eco__img"
+                      src="images/icons/natural.svg"
+                      alt="only natural ingredients">
+
+                    <p class="eco__explanation">
+                      only natural ingredients
+                    </p>
+                  </li>
+                </ul>
+              </div>
+            </div>
+          </section>
+
+          <section class="contact" id="contact">
+            <div class="contact__content">
+              <div class="marker">help</div>
+
+              <div class="contact__information">
+                <h3 class="section__title">
+                  contact us
+                </h3>
+
+                <p class="contact__subtitle">
+                  We will help you choose a product that suits you and answer your other questions
+                </p>
+
+                <form
+                  class="form"
+                  action="#"
+                  method="post"
+                  autocomplete="off"
+                >
+
+                  <input
+                    class="form-field contact__field"
+                    name="name"
+                    type="text"
+                    placeholder="Your Name"
+                    required
+                  >
+
+                  <input
+                    class="form-field contact__field"
+                    name="email"
+                    type="email"
+                    placeholder="E-Mail"
+                    required
+                  >
+
+                  <input
+                    class="form-field contact__field"
+                    name="phone"
+                    type="tel"
+                    placeholder="phone number"
+                    pattern="[0-9]*"
+                    minlength="10"
+                    required
+                  >
+
+                  <textarea
+                    class="
+                      form-field
+                      contact__field
+                      contact__field--last form-field--textarea
```

Comment:
```suggestion
                      contact__field--last 
                      form-field--textarea
```

Reply:
about item 5 - default element without underline. only in hover effect underline is added